src/App.svelte

```diff
@@ -592,6 +598,11 @@
 
     function setSelection(event_or_index) {
         let index = event_or_index.detail?.index ?? event_or_index
+        if (event_or_index.detail?.toBottom) {
+            selection.left = index
+            selectToBottom()
+            return;
```

Comment:
The function can continue here with hijacked parameters, `renderSelection` also gets called at the end of it anyway
```suggestion
            index = chords_and_otherwise.length - 1
```